index.js

```
@@ -0,0 +1,110 @@
+  formulas[0] = mainFormula
+  for (let i = 1; i < 9; i++) {
+    formulas[i] = modifyInfo(mainFormula)
+  }
+}
+
+// Draw the large main pattern and each preview.
+export function updateAll() {
+  const mainCanvas = document.getElementById("mainPattern")
+  drawQbist(mainCanvas, mainFormula, 1)
+  for (let i = 0; i < 9; i++) {
+    const canvas = document.getElementById("preview" + i)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)